README.md

```diff
@@ -50,20 +50,41 @@ _Note:_ The library exposes typings for IDE assistance/auto-complete and compile
 ### Initializing the client
 
 The client will use the OAuth2.0 flow `client_credentials` for authorization. Please refer to these [docs](https://developers.liveperson.com/liveperson-functions-external-invocations-client-credentials.html) for further information on that. On each request the client will check if the `JWT` is about to expire. If this is the case, the client will try to refresh it. If the `JWT` is expired and the client failed to refresh it, an `Error` is thrown. The time after which the refreshing logic will kick in can be specified via the property `jwtRefreshAfterMinutes`.
-Alternatively you can provide your own authorization method that generates a suitable authorization header.
```

Comment:
It would be better if we let that part up here and reformulate it and in the bottom use `// Alternative:` similar to how you did it for the Default option.

Reformulation:
> Alternatively, you can provide your own authorization method that generates a suitable authorization header
> by implementing a method that implements the `GetAuthorizationHeader`-Interface:

```ts
type GetAuthorizationHeader = (input: {
  url: string;
  method: string;
}) => Promise<string>;
```

Reply:
implemented